packages/connectivity/src/scp-cf/identity-service.ts

```diff
@@ -4,12 +4,16 @@ import type { DestinationOptions } from './destination';
 
 /**
  * @internal
- * Make a token exchange from IAS token to XSUAA token.
- * @param options - Configuration for how to retrieve destinations from the destination service.
- * @returns Exchanged token.
+ * Make a token exchange from IAS token to XSUAA token using credentials from destination service.
+ * @param jwt - The IAS JWT to exchange.
+ * @param service - The service used for the token exchange.
+ * @returns Exchanged XSUAA token.
  */
-export async function exchangeToken(jwt: string): Promise<string> {
-  return jwtBearerToken(jwt, 'xsuaa');
+export async function exchangeToken(
```

Comment:
What's the point of this change?
It looks like we exclusively use this function with the destination service, now it has a parameter more and each call needs to add 'destinatin'?

Reply:
Right now we only use this during the destination fetch flow, so the credentials in the destination service are used for the token exchange.
If in the future for some other XSUAA based service if we need to do the exchange, then we could use the respective service credentials for the exchange. 
I did not want the exchange to always rely on the destination service, but rather do it based on the service involved.